docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -0,0 +1,240 @@
+permalink: /documentation/using-testcafe/command-line-interface/
+---
+# Command Line Interface
+
+TestCafe command line interface is pretty simple.
+It consists of a single command that runs tests with a number of options.
+
+## Synopsis
+
+```sh
+testcafe [options] <browser-list-comma-separated> <file-or-glob ...>
+```
+
+## Browser List
+
+### Local Browsers
+
+Specify the list of browsers (separated by commas) where tests will be ran.
+You can specify browser paths or aliases. 
+Aliases are short names TestCafe assigns to browsers for quick reference. 
+A list of all the available aliases can be obtained by the `testcafe --list-browsers` command (see [below](#b)).
+
+The following example demonstrates how to run a test in several browsers. The browsers can be specified by aliases as well as by paths.
+
+```sh
+testcafe chrome,ff,"c:/program files/internet explorer/iexplore.exe" /tests/sample-fixture.js
+```
+
+### All Browsers
+
+To run tests against all installed browsers, use the `all` alias.
+
+```sh
+testcafe all /tests/sample-fixture.js
+```
+
+### <a class="anchor" name="remotes"></a>Remote Browsers
+
+To run tests on a remote browser, specify `remote` as a browser alias. 
+
+```sh
+testcafe remote /tests/sample-fixture.js
+```
+
+TestCafe will provide a URL that you need to open in the desired browser on your remote device. 
+
+You can also use the [--qr-code](#qr-code) option to get a QR-code that represents the same URL. 
+Scan the QR-code by a device on which you are going to test the application.
+As a result, the browser will be connected to TestCafe and tests will start.
+
+If you want to connect multiple browsers,
+prefix alias with the number of browsers you want to connect. For example, if you need to use three remote browsers, specify *3remote*.
+
+```sh
+testcafe 3remote /tests/sample-fixture.js
+```
+ 
+TestCafe will provide an individual URL for each browser.
+Note that tests will start only after all the browsers are connected.
+
+## File Path/Glob Pattern
+
+Specify one or more file paths to run tests from. If you specify several files, separate them by a space.
+
+```sh
+testcafe ie /tests/fixture1.js /tests/fixture2.js
+```
+
+You can use globbing patterns to specify a set of files. 
+For example, the following command runs tests from files that matches `/tests/*page*`, namely `/tests/example-page.js` and `/tests/main-page.js`.
+
+```sh
+testcafe ie /tests/*page*
+```
+
+You can also specify only a directory to run tests from.
+
+```sh
+testcafe ie /my-tests
+```
+
+If you do not specify any file or directory, TestCafe will run tests from `/test` or `/tests` directories.
+
+## Options
+
+### -h, --help  
+Display usage information for commands.
+
+```sh
+testcafe --help
+```
+
+### <a class="anchor" name="b"></a>-b, --list-browsers 
+Lists all browsers available on your system.
+
+```sh
+testcafe --list-browsers
+```
+
+TestCafe assigns the following aliases for browsers installed on your computer:
```

Comment:
It doesn't `assigns` more likely it `uses` or smthg like that. We need to move this to the separate section (see above).